docs/en/engines/table-engines/mergetree-family/summingmergetree.md

```diff
@@ -126,7 +126,7 @@ Table can have nested data structures that are processed in a special way.
 
 If the name of a nested table ends with `Map` and it contains at least two columns that meet the following criteria:
 
-- the first column is numeric `(*Int*, Date, DateTime)` or a string `(String, FixedString)`, let's call it `key`,
+- the first column is numeric `(*Int*, Date, DateTime)` or a string `(String, FixedString)`, or their `LowCardinality` variants, let's call it `key`,
```

Comment:
the first column is numeric `(*Int*, Date, DateTime)`, string `(String, FixedString)` or their `LowCardinality` variants, let's call it `key`